docs/REST API/rest-api.md

```diff
@@ -3536,6 +3592,128 @@ Retrieves a list of the configuration profiles assigned to a host.
 }
 ```
 
+### Get host's past activity
+
+`GET /api/v1/fleet/hosts/:id/activites/past`
```

Comment:
```suggestion
`GET /api/v1/fleet/hosts/:id/activities/past`
```

Reply:
@noahtalerman @marko-lisica do we keep the `/past` suffix? It was spec'd without it.

Reply:
Please ignore, Jahziel told me this had already been addressed in standup (`/past` will not be used).